README.md

```diff
@@ -1,2 +1,57 @@
-# vaccination-stats
-Storing and managing data through TypeQL and Python
+# VACCINATION STATS ONBOARDING_PROJECT
+**Onboarding project to illustrate the connectivity of TypeQL and Python**
+## HOW TO USE
+#### Make sure that you have TypeDB Server installed. (Find instructions [here](https://typedb.com/docs/typedb/2.x/installation)).
+1. Get the repository and extract its files.
```

Comment:
Which repository? Be specific. It would be very easy for a user to get confused and think you mean *this* repository rather than the TypeDB repository. Additionally, if you're cloning the entire repo there's no need to extract files. You'd need to extract files when downloading the release archive from the repo.